packages/salesforcedx-vscode-core/package.json

```diff
@@ -51,7 +51,7 @@
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && shx rm -rf out",
     "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "node ./node_modules/vscode/bin/test"
+    "test": "CODE_TESTS_WORKSPACE='test/sfdx-simple' node ./node_modules/vscode/bin/test"
```

Comment:
Set the workspace env variable before the script runs. This will load the correct workspace into the extension host.